### ISSUE TITLE
restructure code making files smaller

### ISSUE THREAD

**Issue by djay** [djay/covidthailand#93]
- [ ] split covid_data into covid_data_dash.py etc.
- [ ] move related plots into functions. covid_plot.plot_deaths etc

**Comment by djay** [comment]
@reduxionist thoughts?

**Comment by reduxionist** [comment]
I think this would make it much easier to onboard new contributors to the project as it can be hard to puzzle out how things works when you click the outline view and just get 2 functions for covid_plot, versus 50+ for the covid_data module even though the latter is only twice as many SLoC as the former.

It's all working fine as purely procedural code, but are there components of the restructuring you see becoming OOP? Dashboard so often end up being ugly piles of OOP for some reason... :wink:

**Comment by djay** [comment]
The only thing I was thinking of making an object for was to contain all the various dataframes. To make it easier for scrapers to add to multiple frames and easier to pass to the plot. 
Some methods could move to it like import and export and don't think a lot needs to.

**Comment by reduxionist** [comment]
Yeah that sounds reasonable to me if you decide to go there, nothing radical just a structure that some devs may be more comfortable wrapping their brains around. idk. my coding history predates my knowledge of OOP, so to me it's six of one a half-dozen of the other... ;)
